semantic-conventions/src/opentelemetry/semconv/templating/code.py

```diff
@@ -63,10 +63,10 @@ def strong(self, text):
                 return strong.format(text)
             return super().strong(text) if html else text
 
-        def inline_html(self, html_text):
```

Comment:
This also needs to be fixed, since we use `html` already for a boolean 😬

Reply:
Done in 38e978a, @thisthat please take a look again.

Reply:
✔️